Cargo.toml

```diff
@@ -15,6 +15,6 @@ bumpalo = "~3"
 
 [dev-dependencies]
 bumpalo = { version = "~3", features = ["collections"] }
-crossbeam-utils = "~0.7"
+crossbeam-utils = "0.8.11"
```

Comment:
Why is it `0.8.11` and not `0.8`? Is there a specific reason to mandate that at least `.11` patch version instead of just letting the resolution algorithm pick the newest `0.8` available?